### DISCUSSION TITLE
java.util.MissingResourceException in Java Program

### DISCUSSION THREAD

**Discussion by jupiterspot** [community/community#21391] [Other Feature Feedback, Questions, & Ideas]
<p>Hello, </p>
<p>I can’t seem to Run my Java program because of an unknown reason. I have specified my resource bundle variable as:</p>
<pre><code>private static final ResourceBundle bundle = ResourceBundle.getBundle(
        "net.devbase.jfreesteel.viewer.viewer");
</code></pre>
<p>However, it reports that the resource bundle called “viewer” cannot be found. This is how the path to the resource bundle looks like:</p>
<p><img src="https://user-images.githubusercontent.com/27760201/181063997-7e859cc5-e2f2-4cd2-a8bb-b0b9e252275b.png" alt="Capture.PNG" data-base62-sha1="tMgABmEZVjBKZAQsKGZP4ih7hnn" width="304" height="402"></p>
<p>I have tried different combinations in my variable like:</p>
<pre><code>net.devbase.jfreesteel.viewer.viewer
resources.net.devbase.jfreesteel.viewer.viewer
net/devbase/jfreesteel/viewer/viewer
/net/devbase/jfreesteel/viewer/viewer
resources/net/devbase/jfreesteel/viewer/viewer
/resources/net/devbase/jfreesteel/viewer/viewer
</code></pre>
<p>But without luck for some reason the resource bundle cannot be found.</p>
<p>The error message is basically:</p>
<pre><code>java.lang.ExceptionInInitializerError
Caused by: java.util.MissingResourceException: Can't find bundle for base name net.devbase.jfreesteel.viewer.viewer, locale en_US
	at java.util.ResourceBundle.throwMissingResourceException(ResourceBundle.java:1573)
	at java.util.ResourceBundle.getBundleImpl(ResourceBundle.java:1396)
	at java.util.ResourceBundle.getBundle(ResourceBundle.java:782)
	at net.devbase.jfreesteel.viewer.EidViewer.&lt;clinit&gt;(EidViewer.java:81)
Exception in thread "main"
Process finished with exit code 1
</code></pre>
<p>This is not my code. I found the source on github and just wanted to modify a few things. I’m still a Java beginner and didn’t find any similar problem or solution anywhere.</p>
<p>Thanks in advance.</p>

**Comment by fire-eggs** [comment, marked as answer]
<p>Look at <a href="https://stackoverflow.com/questions/2083159/cant-find-bundle-for-base-name" rel="nofollow noopener">this</a> or <a href="https://stackoverflow.com/questions/26752667/cant-find-bundle-for-base-name" rel="nofollow noopener">this</a></p>
